db/seeds/questions.yml

```diff
@@ -174,9 +241,15 @@ why_no_retreat:
   question_type: long_text
   required: false
 
+fa20_heard_from:
+  title: How did you first hear about Blueprint?
+  hint: Whether it’s through social media, through a friend, through classes, through a certain Sproul-alternative, etc.
+  question_category: more_questions
+  question_type: short_text
+
 heard_from:
   title: How did you first hear about Blueprint?
-  hint: Please let us know how you first heard about Blueprint, whether it’s through social media, through a friend, through classes, through Sproul, etc.
+  hint: Whether it’s through social media, through a friend, through classes, through Sproul, etc.
```

Comment:
Replace first 2 words with 'For example,'

I'd also add event or club fair instead of Sproul